lib/SILOptimizer/Transforms/SILCleanup.cpp

```diff
@@ -9,13 +9,18 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
-//
-// Cleanup SIL to make it suitable for IRGen. Specifically, removes the calls to
-// Builtin.staticReport(), which are not needed post SIL.
-//
-// FIXME: This pass is mandatory so should probably be in
-// SILOptimizer/Mandatory.
-//
+///
+/// \file
+///
+/// Cleanup SIL to make it suitable for IRGen.
+///
+/// We perform the following canonicalizations:
+///
+/// 1. We remove calls to Builtin.staticReport(), which are not needed post SIL.
+///
+/// FIXME: This pass is mandatory so should probably be in
+/// SILOptimizer/Mandatory.
```

Comment:
Why not just move it to the mandatory directory

Reply:
I have a chain of patches and I don't want to run into merge issues. I wish I had started it there. But I think git is smart enough now that I can do the move afterwards without messing with the git blame too much.

Reply:
But you know what, maybe git's merge is good enough for this to "commute" the merge with the move.